app/App.tsx

```diff
@@ -152,11 +160,8 @@ class App extends Component {
           <meta property="og:image:width" content="1200" />
           <meta property="og:image:height" content="630" />
         </Helmet>
-        {!isSFFamiliesSite()
-          && config.INTERCOM_APP_ID
-          && <Intercom appID={config.INTERCOM_APP_ID} />}
-        {isSFFamiliesSite()
-          && <UserWay appID={config.SFFAMILIES_USERWAY_APP_ID} />}
+        {userWay && <UserWay appID={config.SFFAMILIES_USERWAY_APP_ID} />}
+        {intercom && config.INTERCOM_APP_ID && <Intercom appID={config.INTERCOM_APP_ID} />}
```

Comment:
Orthogonalized userway and intercom, hopefully this is the way that is wanted.

@fredciaramaglia @jjfreund @richardxia

Reply:
Yep, the way they are now orthogonal looks great!

But I'm also now seeing something a bit weird about these lines, which is that `SFFAMILIES` has still leaked into the line via the `config. SFFAMILIES_USERWAY_APP_ID` configuration setting. This isn't good, because it means that configuration option is actually still tied to the whitelabeled site. At the configuration option level, I think we need to keep the `SFFAMILIES_` prefix because the ID is tied to them and possibly to some subscription they've paid for. However, I'm now thinking that we could maybe handle this within the whitelabel file itself by combining it with the existing `userWay` property.

What do you think of the following idea: In whitelabel.ts, replace the `userWay` property with `userWayAppID`, and have it be `null` either when the whitelabeled site should not have UserWay or when the configuration option is missing? This allows it to play both the role of only enabling it based on which whitelabel site as well as providing the actual value. I think we could do the same thing with Intercom, in which case you could then eliminate the extra check on `config.INTERCOM_APP_ID` here.